R/bootstrap-dea.R

```diff
@@ -0,0 +1,167 @@
+#' @importFrom stats complete.cases median quantile rnorm sd var
+NULL
+
+#' Bootstrap a DEA model
+#'
+#' Run bootstrap on a DEA model to estimate bias corrected efficiency scores and confidence
+#' intervals
+#'
+#' @param dea An object of type pioneer_dea from `compute_dea()`
+#' @param alpha One minus the confidence level required (defaults to 0.05)
+#' @param bw_rule A string with the type of bandwidth rule to be used, or a number with the
+#'   bandwidth parameter. See details.
```

Comment:
Using a number is not an option, so I think this a legacy from older code.

Reply:
Actually it is an option. If you pass a numeric value, this will be used as the bandwidth parameter `h` directly without any estimation based on data. From the `bootstrap_dea_()` internal function:
```r
h <- if (is.numeric(bw_rule)) bw_rule else bw_rule(theta, rule = bw_rule)
```